ui/components/ReconciledObjectsTable.tsx

```diff
@@ -35,23 +35,33 @@ function ReconciledObjectsTable({
   return (
     <div className={className}>
       <DataTable
-        sortFields={["name", "type", "namespace", "status"]}
         fields={[
-          { value: "name", label: "Name" },
+          {
+            value: "name",
+            label: "Name",
+            sortType: SortType.string,
+            sortValue: ({ name }) => name,
+          },
           {
             label: "Type",
             value: (u: UnstructuredObject) => `${u.groupVersionKind.kind}`,
+            sortType: SortType.string,
+            sortValue: (u: UnstructuredObject) => `${u.groupVersionKind.kind}`,
```

Comment:
Not sure why the original was in a string template:

```suggestion
            sortValue: (u: UnstructuredObject) => u.groupVersionKind.kind,
```